### DISCUSSION TITLE
Can't connect with psql to supabase DB

### DISCUSSION THREAD

**Discussion by vavdav** [supabase/supabase#9423] [Questions]
When I am trying to connect to the DB with command `psql -h db.********.supabase.co -p 5432 -d postgres -U postgres` provided in `settings -> database`  I am getting this error:

```
psql: error: connection to server at "db.******.supabase.co" (3.71.175.27), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
```

Any idea how to solve this? I was able to connect once, after supabase project restart, but it does not work any more. I alse tried to use certificate, but still getting this error and I am not sure how to debug it.

**Comment by Sukikiroi** [comment]
Most likely it's because your system shutdown unexpectedly,
Try to Restart your supabase Project again.

**Comment by vavdav** [comment]
It seems that it works again. I wrote `exit` to exit from psql

edit: This seems that it did not help. It is not working after every restart.

**Comment by GaryAustin1** [comment]
I'm not using it but are you getting password prompt or sending the password?
```
-W
--password

    Forces psql to prompt for a password before connecting to a database.

    psql should automatically prompt for a password whenever the server requests password authentication. However, currently password request detection is not totally reliable, hence this option to force a prompt. If no password prompt is issued and the server requires password authentication, the connection attempt will fail.

    This option will remain set for the entire session, even if you change the database connection with the meta-command \connect.
```
Also if you mess up the password too many times (3 I think),  you will be locked out by fail2ban for 30 minutes
